lib/chef/provisioning/docker_driver/driver.rb

```diff
@@ -227,6 +227,7 @@ def machine_for(machine_spec, machine_options, base_image_name)
           :command => docker_options[:command],
           :env => docker_options[:env],
           :ports => [].push(docker_options[:ports]).flatten,
+          :volumes => [].push(docker_options[:volumes]).flatten.compact,
```

Comment:
What happens if `docker_options[:volumes]` isn't set?  i.e. what if the user doesn't pass volumes--it feels like this will probably throw an exception.